client/tracing/src/logging/mod.rs

```diff
@@ -171,6 +171,9 @@ where
 	};
 	let builder = FmtSubscriber::builder().with_env_filter(env_filter);
 
+	#[cfg(not(target_os = "unknown"))]
+	let builder = builder.with_span_events(format::FmtSpan::CLOSE);
```

Comment:
@emostov this change causes [this test](https://github.com/paritytech/substrate/blob/master/client/tracing/src/logging/mod.rs#L387) to fail. Changing it to `FmtSpan::NONE` which seems to work but I honestly don't know what I'm doing here. :/

Reply:
See https://github.com/paritytech/substrate/pull/7780/commits/6e3c3366fbb9137c6391cffa5b8517a4a0ace75b

If anyone finds that controversial feel free to revert